README.md

```diff
@@ -111,37 +110,14 @@ Next run a suite of tests with `make test` and see a corresponding walkthrough
 of API features in [tests/demo/walkthrough.sh](tests/demo/walkthrough.sh). For
 more demonstrations, see the [demos](demos) directory.
 
-## Enabling the Google Cloud Datastore API
-
-For deployments, we recommend using the
-[Google Cloud Datastore API](https://cloud.google.com/datastore). This must be
-enabled for a Google Cloud project and appropriate credentials must be
-available to `registry-server`. One way to get credentials is to use
-[Application Default Credentials](https://cloud.google.com/docs/authentication/production).
-To get set up, just run `gcloud auth application-default login` and sign in.
-Then make sure that your project id is set to the project that is enabled to
-use the Google Cloud Datastore API.
-
-Notes:
-
-- You only need to get credentials when you are running the server locally.
-  When `registry-server` is run with Google Cloud Run, credentials are
-  automatically provided by the environment.
-- When enabling the Datastore API, you might be asked to select a storage mode.
-  This project's Datastore API usage is equivalent to
-  [running Cloud Firestore in Datastore mode](https://cloud.google.com/datastore/docs).
-- The reference implementation requires indexes in its Datastore instance. To
-  create these indexes, use the `gcloud` command in the root of this
-  repository: `gcloud datastore indexes create server/datastore/index.yaml`
-
 ## Running the Registry API server locally
 
 ### Running the Registry API server
 
 Running `source auth/LOCAL.sh` will configure your environment to run the
 Registry API server locally and for the included clients to call your local
-instance. Start the server by running `registry-server`. (Recall that by
-default, this uses the Cloud Datastore API, a remote service).
+instance. Start the server by running `registry-server -c config/postgres.yaml`
```

Comment:
I think a link makes the most sense, but it required a few extra lines of code to read the file since ioutil.ReadFile doesn't follow symlinks. 

Also, I didn't find a way to make this default work independent of the working directory for the user starting the server, i.e. this default only works if you run the server from the root directory of the project repository. Is this a concern to you? If you have an idea for an alternate solution please let me know.

Reply:
We can hard-code a default that's used when the registry.yaml isn't specified. I think a SQLite database at /tmp/registry.db would be ok for this.

Reply:
Updated this to start the server with the default config. Also updated the default to be a hardcoded SQLite config.